port/zos390/omrportpg.h

```diff
@@ -93,4 +105,6 @@ typedef struct OMRPortPlatformGlobals {
 #define PPG_si_executableName (portLibrary->portGlobals->platformGlobals.si_executableName)
 #define PPG_ipt_ttoken (portLibrary->portGlobals->platformGlobals.iptTtoken)
 
+#define PPG_stfleCache (portLibrary->portGlobals->platformGlobals.stfleCache)
```

Comment:
Should this macro be wrapped under an OMR_ENV_DATA64 ifdef?

Reply:
Similarly there was no ifdef in the openJ9 equivalent